grails-app/views/inventoryItem/_showStockHistory.gsp

```diff
@@ -340,6 +340,26 @@
 
                                 <g:if test="${stockHistoryEntry?.showDetails}">
                                     <div >
+                                        <g:if test="${stockHistoryEntry?.transaction?.order?.code == org.pih.warehouse.order.OrderTypeCode.PURCHASE_ORDER.name()}">
+                                            <g:link controller="stockMovement" action="show" id="${stockHistoryEntry?.transaction?.order?.id }">
+                                                <div title="${stockHistoryEntry?.transaction?.order?.name }">
+                                                    <format:metadata obj="${stockHistoryEntry?.transaction?.order?.orderType?.code }"/>
+                                                    &rsaquo;
+                                                    ${stockHistoryEntry?.transaction?.order?.orderNumber }
+                                                </div>
+                                            </g:link>
+                                        </g:if>
+
+                                        <g:elseif test="${stockHistoryEntry?.transaction?.order?.code == org.pih.warehouse.core.Constants.RETURN_ORDER}">
```

Comment:
You can import `Constants` too, same as `OrderTypeCode`

Reply:
done